cmd/mr_approve.go

```diff
@@ -59,15 +51,25 @@ var mrApproveCmd = &cobra.Command{
 				os.Exit(1)
 			}
 		}
+
+		if comment || len(msgs) > 0 || filename != "" {
```

Comment:
Maybe it's worth adding a `strings.TrimSpace(filename)`  here?

Reply:
ah yea.. the same would be valid in the mr_unapprove.go

Reply:
You're right, a new commit with the added trim has pushed :)